pytest/tests/mocknet/mirror.py

```diff
@@ -347,6 +348,21 @@ def update_binaries_cmd(args, traffic_generator, nodes):
          nodes + [traffic_generator])
 
 
+def run_remote_cmd(args, traffic_generator, nodes):
+    targeted = []
+    if args.all or args.traffic:
+        targeted.append(traffic_generator)
+    if args.all or args.nodes:
+        targeted.extend(nodes)
+    if args.filter is not None:
+        targeted = [h for h in targeted if re.search(args.filter, h.name())]
+    if len(targeted) == 0:
+        logger.error(f'No hosts selected. Change filters and try again.')
+        return
+    logger.info(f'Running cmd on {"".join([h.name() for h in targeted ])}')
+    pmap(lambda node: cmd_utils.run_cmd(node.node.node, args.cmd), targeted)
```

Comment:
this will not work if you are running a mocknet locally with the setup in `local_test_node.py`, and then using mirror.py with the `--local-test` flag, because then `node.node.node` is not the object you thought it would be. Maybe you could do this by adding a `run_cmd()` function to `NodeHandle` in `node_handle.py` that calls yet another `run_cmd()` function defined in the objects in both `remote_node.py` and `local_test_node.py`, where the latter will do nothing and maybe just print out some info

Reply:
Done.